src/js/models/panel_model.js

```diff
@@ -368,6 +368,8 @@
             y_size = this.get('pixel_size_y');
             z_size = this.get('pixel_size_z');
             z_size = z_size?z_size:0
+            x_size = x_size?x_size:0
+            y_size = y_size?y_size:0
```

Comment:
Minor point: I find that kinda hard to read without spaces. Could you use `y_size = y_size ? y_size : 0;`
Elsewhere I've started to use `Prettier` for code formatting to handle these cases, but not here yet... thx.

Reply:
yes no problem. I'll go refactor it where I find it